paperqa/contrib/zotero.py

```diff
@@ -216,7 +229,33 @@ def iterate(
             num_remaining = limit - actual_i
 
         self.logger.info("Finished downloading papers. Now creating Docs object.")
+        
```

Comment:
```suggestion
        def _sliced_collection_items(self, collection_id, limit, start):
            items = self.collection_items(self, collection_id)
            items = items[start:]
            if len(items) < limit:
                return items
            return items[:limit]
```

Reply:
Thanks again for your suggestions!! (That's my fault. I misunderstood...)